docs/userguide/modules/getting-started/pages/petclinic-pgcluster.adoc

```diff
@@ -0,0 +1,73 @@
+= Connecting SpringBoot application to operator-backed PostgreSQL database
```

Comment:
should we be here more specific - the section is about connection PetClinic application to operator-backed PostgresSQL database?

Reply:
sure... will update it